packages/server/src/api/controllers/automation.js

```diff
@@ -229,35 +144,34 @@ exports.find = async function (ctx) {
 
 exports.destroy = async function (ctx) {
   const db = new CouchDB(ctx.appId)
-  const oldAutomation = await db.get(ctx.params.id)
+  const automationId = ctx.params.id
+  const oldAutomation = await db.get(automationId)
   await checkForWebhooks({
     appId: ctx.appId,
     oldAuto: oldAutomation,
   })
-  await checkForCronTriggers({
-    appId: ctx.appId,
-    oldAuto: oldAutomation,
-  })
-  ctx.body = await db.remove(ctx.params.id, ctx.params.rev)
+  // delete metadata first
+  await cleanupAutomationMetadata(ctx.appId, automationId)
+  ctx.body = await db.remove(automationId, ctx.params.rev)
 }
 
 exports.getActionList = async function (ctx) {
-  ctx.body = actions.DEFINITIONS
+  ctx.body = actions.ACTION_DEFINITIONS
 }
 
 exports.getTriggerList = async function (ctx) {
-  ctx.body = triggers.BUILTIN_DEFINITIONS
+  ctx.body = triggers.TRIGGER_DEFINITIONS
 }
 
 exports.getLogicList = async function (ctx) {
-  ctx.body = logic.BUILTIN_DEFINITIONS
+  ctx.body = logic.LOGIC_DEFINITIONS
```

Comment:
Hey @mike12345567 I see here that Logic is still an object of its own, but in the designs it seemed like they're now actions (https://xd.adobe.com/view/e4f6d4aa-86f3-48ca-a92a-b6d3d70b5a75-b1a5/screen/341effa1-ad27-42a8-af01-e68f677282a0?fullscreen) and i think @joebudi might have mentioned this as well to me at some point.

Reply:
Ah, do you want me to represent that in the backend as well? I had just left it as it was because the API returns them altogether anyway, didn't want to break the API endpoint for you!

Reply:
I think that would maybe be cleaner, because I would have to stitch them into the actions object in the frontend anyways. But it's no big deal either way, if it's hassle in any way at all I can easily just bring them together myself.